anatomy/tracks/concept-exercises.md

```diff
@@ -276,6 +276,7 @@ This file contains meta information on the exercise:
   - `test`: the [test file(s)](./#filetests) (required)
   - `exemplar`: the [exemplar implementation file(s)](./#fileexemplarimplementation) (required)
 - `language_versions`: Language version requirements (optional)
+- `blurb`: A short description of this exercise (required)
```

Comment:
@iHiD What is the maximum length of the blurb?

Reply:
🤷 Could you see what the current max length is in problem specs pls?

Reply:
```
16
20
23
24
25
25
26
26
26
27
28
28
30
30
30
31
33
33
34
35
35
35
36
36
36
37
38
38
38
41
41
42
42
43
44
44
44
44
46
46
46
47
47
47
49
49
50
50
50
51
51
51
52
54
55
55
56
57
57
57
58
60
60
61
61
61
61
61
62
62
65
66
66
67
67
69
69
69
69
69
70
71
71
72
73
73
74
75
75
78
80
80
80
80
80
81
81
81
81
83
85
90
90
91
91
92
92
95
95
96
96
100
100
101
101
103
105
108
108
114
114
116
118
119
121
124
124
130
136
142
150
202
207
241
309
```

Those are all the blurb lengths

Reply:
OK, let's go with 350 chars, which is the same as the track concept bios.

Reply:
:+1

Reply:
Updated